cosmos/providers/dbt/task_group.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         emit_datasets: bool = True,
         dbt_root_path: str = "/usr/local/airflow/dbt",
         test_behavior: Literal["none", "after_each", "after_all"] = "after_each",
-        dbt_tags: List[str] = [],
+        select: Dict[str, Any] = {},
```

Comment:
could/should this be something like:
```suggestion
        select: Dict[str, List[str]] = {},
```

Reply:
Added.